core/templates/dev/head/pages/exploration_editor/translation_tab/audio_translation_bar_directive.html

```diff
@@ -175,88 +205,168 @@
     width: 100%;
   }
   .oppia-audio-recording-bar {
-    width: 100%;
+    align-items: center;
+    background-color: #f7f7f7;
+    display: flex;
     height: 100%;
-    display: inline-block;
+    padding: 0px 10px;
+    width: 100%;
+  }
+  .audioRecorder {
+    align-items: center;
+    display: flex;
+  }
+  .audioRecorder > div:first-child {
+    width: 0 !important;
+  }
+  .oppia-content-wrapper {
+    margin: 0 10px;
   }
   .oppia-audio-bar-info {
-    float: left;
-    width: 79%;
+    background-color: #009688;
+    border-radius: 3px;
+    color: white;
+    flex: 1;
+    font-size: 12px;
+    height: 38px;
+    line-height: 27px;
+    padding: 5px 40px;
     text-align: center;
-    padding: 13px 0px 13px 0px;
-    border-left: 1px solid #ccc;
-    border-right: 1px solid #ccc;
   }
   .oppia-audio-button {
-    background: white;
-    color: #009688;
-    float: left;
-    border-left: 1px solid #ccc;
-    border-right: 1px solid #ccc;
-    border-bottom: 1px solid #ccc;
-    border-radius: 0px;
-    padding: 13px;
-    width: 7%
-  }
-  .oppia-audio-visulaizer {
-    width: 72%;
-    float: left;
-    border-left: 1px solid #ccc;
-    border-right: 1px solid #ccc;
+    background-color: #009688;
+    border-radius: 20px;
+    color: white;
+    padding-left: 6px;
+    transition: background-color .2s cubic-bezier(.35,0,.25,1);
+    width: 38px;
+  }
+  .oppia-retake-button {
+    background-color: #009688;
+    border-radius: 3px;
+    color: white;
+    padding-left: 6px;
+    transition: background-color .2s cubic-bezier(.35,0,.25,1);
+    width: 63px;
+  }
+  .oppia-confirm-button {
+    background-color: #009688;
+    border-radius: 3px;
+    color: white;
+    padding-left: 6px;
+    transition: background-color .2s cubic-bezier(.35,0,.25,1);
```

Comment:
Create a new class `oppia-audio-bar-button-transition` and move this transition into it with support for other browsers (also test this with firefox.)

Reply:
done

Reply:
Try adding these for cross-browser support:
`-webkit-...  // For Webkit browser(Chrome, Safari...)`
`-moz-...     // For Mozilla browser`
`-o-...       // For Opera browser`

Reply:
done! @DubeySandeep :)
PTAL!